packages/website/docs/docs/side-menu.md

```diff
@@ -177,6 +177,10 @@ type SideMenuButtonProps = {
 }
 export const SideMenuButton = (props: SideMenuButtonProps) => ...;
 
-// Takes same props as `button` elements, e.g. onClick.
-export const DragHandleMenuItem = (props) => ...;
+// Also includes all props of regular `button` elements, e.g. onClick.
```

Comment:
this comment conflicts with the type definition below. I'd either:
a) make the type definition complete
b) put the comment inside the `= { ... }`
c) point to type definition in code